app/assets/javascripts/govuk_publishing_components/rum-loader.js.erb

```diff
@@ -0,0 +1,36 @@
+(function() {
+  var marker = document.getElementsByTagName('script')[0]
+  var scripts = [
+    '<%= path_to_javascript("govuk_publishing_components/vendor/lux/lux-polyfill") %>',
+    '<%= path_to_javascript("govuk_publishing_components/vendor/lux/lux") %>'
+  ]
+  var parsedCookie = (function() {
```

Comment:
Can we use any of the existing cookie functions code for any of this or does it not offer this functionality?

Reply:
This needs to run as quickly as possible without waiting for anything else to load - parsing the cookie here means no dependencies to wait for. 

The cookie helper functions are bundled into `application.js`, which is loaded just before the closing body tag - whilst we could move the specific cookie helper that reads cookies out of that bundle, it just felt as if it was asking for trouble further down the line.